python/geom.py

```diff
@@ -234,6 +234,27 @@ def transform(self, m):
 
         for s in self.H_susceptibilities:
             s.transform(m)
+    def get_eps(self,freq):
```

Comment:
Since we'll probably also want a `get_mu`, maybe this should be refactored into a `_get_epsmu(tensor, susceptibilities, conductivity)` function that is called by `get_eps` and `get_mu`

And maybe the functions should just be called `epsilon(self, freq)` and `mu(self, freq)`?

Reply:
I think this is a good idea.

Where would you prefer that `-get_epsmu()` reside? Would it return both `eps` and `mu`?

Reply:
That is, you would have
```
_get_epsmu(diag, offdiag, susceptibilities, conductivity_diag, conductivity_offdiag, freqs)
```
and from `epsilon` you would call `_get_epsmu(self.epsilon_diag, self.epsilon_offdiag, self.E_susceptibilities, self.D_conductivity_diag, self.D_conductivity_offdiag, freqs)` whereas for `mu` you would pass the corresponding magnetic quantities.

Also, I think it would be convenient to return a number rather than a list if `freqs` is a number.